virttest/utils_test/qemu/__init__.py

```diff
@@ -505,6 +506,23 @@ def get_guest_free_mem(cls, vm):
         finally:
             session.close()
 
+    @classmethod
+    def get_guest_crashkernel_size(cls, vm):
+        """
+        Linux guest OS reported crashkernel size in MB.
+
+        :param vm: VM Object
+        :return: crashkernel report by guest OS in MB
+        """
+        timeout = float(vm.params.get("login_timeout", 600))
+        session = vm.wait_for_login(timeout=timeout)
+        try:
+            crashkernel = session.cmd_output("cat /sys/kernel/kexec_crash_size")
+            crashkernel = utils_misc.normalize_data_size(crashkernel, order_magnitude="M")
+            return math.ceil(float(crashkernel))
+        finally:
+            session.close()
+
```

Comment:
`return 0` if the try block fails? So `guest_mem_size += crashkernel_size` will be safer.